extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -92,6 +92,13 @@ public class OidcTenantConfig extends OidcCommonConfig {
     @ConfigItem
     public Optional<String> endSessionPath = Optional.empty();
 
+    /**
+     * The paths which must be secured by this tenant. Path-matching is done exactly in a same manner as is done
+     * for the 'quarkus.http.auth.permission.perm-1.paths' configuration property paths.
```

Comment:
@michalvavrik IMHO having something like `Path-matching is done exactly in a same manner as is done
     for the 'quarkus.http.auth.permission.perm-1.paths' configuration property paths` is complicating things. What if the users have never used those policies, and don't know what it is ?

Perhaps you can show an example here of two tenants, each having a single path and explaining when each tenant wins ? Users don't need to know all the details at this point, it will be explained in the security docs

Reply:
> @michalvavrik IMHO having something like `Path-matching is done exactly in a same manner as is done for the 'quarkus.http.auth.permission.perm-1.paths' configuration property paths` is complicating things. What if the users have never used those policies, and don't know what it is ?

Personally, I'd go to page with all configuration properties and look it up if didn't know. Brevity is sometimes clearer because reading long texts is discouraging.

> Perhaps you can show an example here of two tenants, each having a single path and explaining when each tenant wins ? 

I don't mind, but it will be like 2 scenarios out of tens of possible combinations.

> Users don't need to know all the details at this point, it will be explained in the security docs

I think it is very hard to extract path matching from current _Securing Webendpoints ..._ guide. I don't say I won't do your suggestions, but are you use it is really not clear? In docs there is a link and when you click on the link, you can see how it works. We didn't have complaints about that path-matching sections...

Reply:
@michalvavrik What I'm proposing is not to refer to something like `'quarkus.http.auth.permission.perm-1.paths'` in the JavaDocs for this property - it won't make sense for users who haven't worked with HTTP Policy config. HTTP Policy config is not even relevant in this context

Reply:
The fact HTTP Policy config and this property share the same matching mechanism should not be part of thi property's JavaDocs

Reply:
Okay